components/datadog/updater/install_script.sh

```diff
@@ -82,6 +84,14 @@ if [ "${OS}" = "Debian" ]; then
 
     $sudo_cmd DEBIAN_FRONTEND=noninteractive apt-get update
     $sudo_cmd apt-get install -y --force-yes datadog-installer
+
+    # Add packages
+    $sudo_cmd systemctl stop datadog-agent
+    for pkg in $PACKAGES; do
+        $sudo_cmd $INSTALLER_BIN bootstrap --url "${OCI_URL_PREFIX}${pkg}:latest"
```

Comment:
I think it'd be better if we passed ${pkg:version} as parameter instead of always using latest since you might want to test a new package without having to push a new OCI tagged lattes